TezosKit/Operation/OperationFactory.swift

```diff
@@ -0,0 +1,113 @@
+// Copyright Keefer Taylor, 2019.
+
+import Foundation
+
+/// A factory which can produce operations.
+public class OperationFactory {
```

Comment:
Activation operation should be added

Reply:
I deprecated the activation operation. TezosKit never properly supported it and with a large percentage of fundraiser allocations activated, it's not a priority to implement.